packages/nvidia_nat_mcp/src/nat/plugins/mcp/server/routes.py

```
@@ -0,0 +1,190 @@
+                # Collect configured workflow tools
+                configured_short_names: set[str] = set()
+                configured_full_to_fn: dict[str, Function] = {}
+                try:
+
+                    async def pass_through_filter(fn):
+                        return fn
+
+                    accessible_functions = await group_instance.get_accessible_functions(filter_fn=pass_through_filter)
+                    configured_full_to_fn = accessible_functions
+                    configured_short_names = {name.split('.', 1)[1] for name in accessible_functions.keys()}
+                except Exception as exc:  # noqa: BLE001
```

Ruff (0.14.13)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)